templates/register.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{% from 'bootstrap5/form.html' import render_form, render_field %}
+{% block app_content %}
+    <h1>Регистрация</h1>
+    <div class="container">
+            <form action="" method="post">
+                {{ form.csrf_token() }}
+                {{ render_field(form.username) }}
+                {{ render_field(form.email) }}
+                {{ render_field(form.password) }}
+                <p><input type=submit value=Register></p></form>
```

Comment:
Эта кнопка уже есть в форме
```suggestion
                <p>{{ form.submit() }}</p></form>
```